single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -305,8 +337,26 @@ private void handleSwitchPositionchange(Container c, String switchId) {
 
         private void loadDiagram(Container c) {
             WebView diagramView = new WebView();
+            diagramView.setZoom(1.0);
             ContainerDiagramResult result = createContainerDiagramView(diagramView, c);
             flowPane.setContent(diagramView);
+            // Adjust size to content
+            diagramView.prefHeightProperty().bind(flowPane.heightProperty());
+            diagramView.prefWidthProperty().bind(flowPane.widthProperty());
```

Comment:
strange you need to do that, could you remove it? I think it comes from the unnecessary scrollPane parent